applications/ecallmgr/src/ecallmgr_util.erl

```diff
@@ -59,7 +59,7 @@
 
 -define(HTTP_GET_PREFIX, "http_cache://").
 
--define(FS_MULTI_VAR_SEP, ecallmgr_config:get_ne_binary(<<"multivar_separator">>, <<";">>)).
+-define(FS_MULTI_VAR_SEP, ecallmgr_config:get_ne_binary(<<"multivar_separator">>, <<"|">>)).
```

Comment:
@sergey-safarov thanks, you'll need to change the default in `system_config.ecallmgr.json`  [here](https://github.com/2600hz/kazoo/pull/4992/files#diff-ad6fddb22739cbba2b2a453234de7a2dR262) and run `make apis` to update swagger